server/feed/views.py

```diff
@@ -215,5 +215,8 @@ def delete(self, request, post_id, comment_id):
         comment = Comment.objects.get(id=comment_id, feed_id=post_id)      
         if comment.user == request.user:
             comment.delete()
+            post = Post.objects.get(id=post_id)
```

Comment:
Can we use django transaction here? So that if Comment delete query failed it shouldn't decrement post count.

Reply:
Please check the latest commit of this PR @shahmedme